src/Schema/Directives/AllDirective.php

```diff
@@ -60,7 +60,9 @@ public function resolveField(FieldValue $fieldValue): callable
                     $query instanceof QueryBuilder || $query instanceof EloquentBuilder || $query instanceof ScoutBuilder || $query instanceof Relation,
                     "The method referenced by the builder argument of the @{$this->name()} directive on {$this->nodeName()} must return a Builder or Relation."
                 );
+                /** @var QueryBuilder|EloquentBuilder<\Illuminate\Database\Eloquent\Model>|ScoutBuilder|Relation<\Illuminate\Database\Eloquent\Model> $query */
             } else {
+                /** @var EloquentBuilder<\Illuminate\Database\Eloquent\Model> $query */
```

Comment:
please revert this, I would like to explore different ways of fixing any error here